src/components/Profile/Profile.jsx

```diff
@@ -39,6 +46,7 @@ export default function InsetDividers() {
                 <CardMedia
                   className={classes.media}
                   image="https://miro.medium.com/max/1200/1*mk1-6aYaf_Bes1E3Imhc0A.jpeg"
```

Comment:
Download the image and import it in the project instead of using external links.